src/client/dfs/SConscript

```diff
@@ -21,7 +21,7 @@ def scons():
             gotversion = True
     if gotversion is True:
         print("Building with Lustre bindings.")
-        denv.AppendUnique(CCFLAGS=['-DLUSTRE_INCLUDE'])
+        conf.env.AppendUnique(CCFLAGS=['-DLUSTRE_INCLUDE'])
```

Comment:
why this change?   denv looks right here

Reply:
@bfaccini could you please address that?

Reply:
I guess maybe it's fine but to be consistent with every other use of configure, it may be better to just use denv here and not set denv = conf.Finish()

Reply:
Well, my understanding is that conf.env is like clone of the current Environment when reaching Configure(), where it can be modified and then passed-back/returned using conf.Finish() ... BTW, it is always stated that Finish() has to be done, right ?

Reply:
Yeah, we always do Finish.   I just don’t see a difference between the two approaches of modifying the Configure environment and modifying the external one directly.  They should both have exactly the same effect.   I’m fine either way really but just saying we do configure checks in a lot of other places and all call Finish without capturing the environment